beacon-node-oapi.yaml

```diff
@@ -95,6 +95,8 @@ paths:
     $ref: "./apis/beacon/blocks/root.yaml"
   /eth/v1/beacon/blocks/{block_id}/attestations:
     $ref: "./apis/beacon/blocks/attestations.yaml"
+  /eth/v1/beacon/blobs_sidecars/{block_id}:
```

Comment:
I'd actually be inclined to make this the same pattern as attestations, so the path would be more like
`/eth/v1/beacon/blocks/{block_id}/blobs`

Reply:
Wouldn't that imply that the blobs are part of the block structure, though?

Reply:
Updated to `/eth/v1/beacon/blobs/{block_id}`